res/controllers/Pioneer-DDJ-FLX4-script.js

```diff
@@ -0,0 +1,779 @@
+// Pioneer-DDJ-FLX4-script.js
+// ****************************************************************************
+// * Mixxx mapping script file for the Pioneer DDJ-FLX4.
+// * Mostly adapted from the DDJ-400 mapping script
+// * Authors: Warker, nschloe, dj3730, jusko, Robert904
+// * Reviewers: Be-ing, Holzhaus
```

Comment:
```suggestion
```
These were the reviewers for the DDJ-400 mapping script -> Just remove this line